.github/styles/alex/Race.yml

```diff
@@ -0,0 +1,87 @@
+---
+extends: substitution
+message: Consider using '%s' instead of '%s'.
+ignorecase: true
+level: warning
+action:
+  name: replace
+swap:
+  Gipsy: Nomad|Traveler|Roma|Romani
+  Indian country: enemy territory
+  animal spirit: favorite|inspiration|personal interest|personality type
+  black list: blocklist|wronglist|banlist|deny list
+  blacklist: blocklist|wronglist|banlist|deny list
+  blacklisted: blocklisted|wronglisted|banlisted|deny-listed
+  blacklisting: blocklisting|wronglisting|banlisting|deny-listing
+  bugreport: bug report|snapshot
+  circle the wagons: defend
+  dream catcher: favorite|inspiration|personal interest|personality type
+  eskimo: Inuit
+  eskimos: Inuits
+  ghetto: projects|urban
+  goy: a person who is not Jewish|not Jewish
+  goyim: a person who is not Jewish|not Jewish
+  goyum: a person who is not Jewish|not Jewish
+  grandfather clause: legacy policy|legacy clause|deprecation policy
+  grandfather policy: legacy policy|legacy clause|deprecation policy
+  grandfathered: deprecated
+  grandfathering: deprecate
+  gyp: Nomad|Traveler|Roma|Romani
+  gyppo: Nomad|Traveler|Roma|Romani
+  gypsy: Nomad|Traveler|Roma|Romani
+  hymie: Jewish person
+  indian give: "go back on one\u2019s offer"
+  indian giver: "go back on one\u2019s offer"
+  japs: Japanese person|Japanese people
+  jump the reservation: disobey|endure|object to|oppose|resist
+  latina: Latinx
+  latino: Latinx
+  long time no hear: "I haven\u2019t seen you in a long time|it\u2019s been a long\
+    \ time"
+  long time no see: "I haven\u2019t seen you in a long time|it\u2019s been a long\
+    \ time"
+  master: primary|hub|reference
+  masters: primaries|hubs|references
+  mexican: Latinx
+  natives are becoming restless: dissatisfied|frustrated
+  natives are getting restless: dissatisfied|frustrated
+  natives are growing restless: dissatisfied|frustrated
+  natives are restless: dissatisfied|frustrated
+  non white: person of color|people of color
+  nonwhite: person of color|people of color
+  off reserve: disobey|endure|object to|oppose|resist
+  off the reservation: disobey|endure|object to|oppose|resist
+  on the warpath: defend
+  oriental: Asian person
+  orientals: Asian people
+  pinays: Filipinos|Filipino people
+  pinoys: Filipinos|Filipino people
+  pocahontas: Native American
+  pow wow: conference|gathering|meeting
+  powwow: conference|gathering|meeting
+  primitive: simple|indigenous|hunter-gatherer
+  red indian: Native American
+  red indians: Native American People
+  redskin: Native American
+  redskins: Native American People
+  sand niggers: Arabs|Middle Eastern People
+  savage: simple|indigenous|hunter-gatherer
```

Comment:
what if "savage" not used to describe hunter-gatherers (overwhelmingly more common)? As in "my savage review of `.github/styles`?"

Reply:
😆 Ok.  *Definite* issues here.  But can't we just have it run Grammarly?  Hard stop.  

That's all we need in my opinion.  That's what is the big time suck to all involved. 

Per what Jorge put above.  We may have to develop our own styles (which will be a project and take time).

Reply:
> what if "savage" is not used to describe hunter-gatherers (overwhelmingly more common)? As in "my savage review of .github/styles?"

@casperdcl All your comments make sense, those are rules built into the default packages provided. The reason for having `.github/styles` is to be able to modify/remove rules that don't apply, like your comments in this P.R, by just removing the `yaml` file.

Reply:
>  But can't we just have it run Grammarly?

@jendefig The short answer is no / not easy to run on the GitHub P.R.

> That's all we need in my opinion. That's what is the big time suck to all involved.

The best I can suggest is trying to edit the blog posts in VSCode and use the [Grammarly](https://marketplace.visualstudio.com/items?itemName=znck.grammarly) and [GitHub](https://code.visualstudio.com/docs/editor/github) extensions. 

This requires some investment in getting familiar with the editor and used to the new workflow. 

However, after setup you are able to view/respond to GitHub comments and get Grammarly suggestions in the same place you edit/write the blog:

<img width="1792" alt="Captura de Pantalla 2022-09-20 a las 10 17 07" src="https://user-images.githubusercontent.com/12677733/191205669-d7173710-d98e-41dd-8b00-ee30dcf21f06.png">

Reply:
Thanks for the recommendation @daavoo !

Reply:
This is great!  Thanks for pointing this out @daavoo!!!! 🙏 I have installed it in VS Code. It's asking me to log in, and I have, with the same Gmail as GitHub, but it's not giving the automated fix for the issue, but the suggestion at least so that's progress!

Reply:
@jendefig I don't know how to sync accounts, I am using the anonymized (no login) version. 

I can click on `Quick Fix` at the bottom left (or press the shortcut Comand + `.` if the cursor is on top of the error) to see and apply suggestions. 

It is still a worse UX than the web, but it's something. Maybe syncing the account enables real auto-fixing?